src/main/java/jalau/at18/katas/pokerhands/mauricio/StraightFlushComparer.java

```diff
@@ -0,0 +1,8 @@
+package jalau.at18.katas.pokerhands.mauricio;
+
+public class StraightFlushComparer {
```

Comment:
Looks like actually, you could have an abstraction `Comparer` where you apply dependency inversion principle by making that class depend on `IdentifyHand` instead of concrete `StraightFlushIdentifier`, `FlushIdentifier`, etc.